frontend/lib/routes/company/CompanyTable.dart

```diff
@@ -97,14 +80,14 @@ class _MemberCompaniesRowState extends State<MemberCompaniesRow>
   Member member;
   String filter;
   CompanyService _companyService = CompanyService();
-  late Future<List<Company>> _companies;
+  late Future<List<CompanyLight>> _companies;
```

Comment:
CompanyLight does not contain participation info, therefore no status to be filtered by. CompanyLight cannot be used here